docs/manual/src/udl/structs.md

```diff
@@ -2,6 +2,8 @@
 
 Dictionaries can be compared to POJOs in the Java world: just a data structure holding some data.
```

Comment:
Oh dear 😅 

Honestly, "compared to POJO" and "compared to JavaBeans" both do absolutely nothing to help my personal understanding of what's happening here, which is maybe a signal that we should just reword this to use a general description rather than a language-specific analogy. Thanks for pointing it out.